client/src/app/users/user.component.html

```diff
@@ -1,7 +1,11 @@
-<div *ngIf="user" class="user">
-  <ul>
-    <li>Name: {{ user.name }}</li>
-    <li>Age: {{ user.age }}</li>
-    <li>Company: {{ user.company }}</li>
-  </ul>
-</div>
+<mat-expansion-panel *ngIf="user" class="user" matTooltip="email: {{user.email}}" #elem [id]="user.email">
+
+  <mat-expansion-panel-header>
+    <mat-panel-title>{{ user.name }}</mat-panel-title>
+  </mat-expansion-panel-header>
+
+  <mat-panel-description>Name: {{ user.name }}</mat-panel-description>
```

Comment:
`mat-panel-description` is used in the `mat-expansion-panel-header`, in the body of the expansion panel it should just use `p` elements or similar.

Reply:
This could explain why we've had students have a tough time with their accordions over the past few years. I vaguely recall that there has been some difficulty with figuring out how to display things or how to maybe it was... test for things? Maybe the description is not found in E2E tests or something. Sigh. Good to know.